api/tests/unit/domain/usecases/correct-answer-then-update-assessment_test.js

```diff
@@ -129,6 +130,25 @@ describe('Unit | Domain | Use Cases | correct-answer-then-update-assessment', fu
     });
   });
 
+  context('when the assessment has been ended because session was finalized', function () {
+    it('should throw a CertificationEndedByFinalization error', async function () {
+      // given
+      assessment.state = Assessment.states.ENDED_BY_FINALIZATION;
```

Comment:
:scream: 
De la mutation d'un objet instantié dans un `beforeEach`
```suggestion
	   const assessment = domainBuilder.buildAssessment({ userId, lastQuestionDate: nowDate });
	   assessment.endByFinalization();
```

Reply:
Vu avec @laura-bergoens, c'est ce qui était fait au départ mais ça suggère trop que le test connait l'implem